### PR TITLE
Move to version 5.0.0

### PR DESCRIPTION
**Pull request by bernaferrari**
cc @jakemac53

### CODE REVIEW

**Review by jakemac53 — APPROVED** [review]

**Comment by jakemac53** [comment]
cc @gspencergoog (I don't have merge permissions here). Can you also publish this once it lands?

**Comment by gspencergoog** [comment]
Hmm. I tried publishing, but I got a warning about it depending on a prerelease version of Dart.

Created https://github.com/google/process.dart/pull/79 to remedy that before publishing.

**Comment by gspencergoog** [comment]
Published to pub

**Pull request merged**
